public/app/plugins/datasource/tempo/datasource.ts

```diff
@@ -55,12 +58,16 @@ export class TempoDatasource extends DataSourceWithBackend<TempoQuery, TempoJson
   serviceMap?: {
     datasourceUid?: string;
   };
+  search?: {
+    hide?: boolean;
```

Comment:
Out of curiosity, why do we need the option to hide the search tab? Is it because we're planning to make it selected by default and users without switch would need to change it for every query?